src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -127,13 +127,13 @@ export class ExposureNotificationService {
     return addDays(cycleStart ? new Date(parseInt(cycleStart, 10)) : new Date(), 14);
   }
 
-  async updateExposureStatus(): Promise<ExposureStatus> {
+  async updateExposureStatus(forceRefresh: boolean = false): Promise<ExposureStatus> {
```

Comment:
I think we should have a better way to abstract `forceRefresh`. In general, we don't want any code / logic in Test mode available in production code. I will look into it sometime today.